includes/definitions/discovery/saf-cfml4.yaml

```diff
@@ -0,0 +1,423 @@
+--- 
+mib: SAF-MPMUX-MIB
+modules: 
+  processors:
+    data:
+      -
+        num_oid: '.1.3.6.1.4.1.7571.100.1.1.2.22.1.28.{{ $index }}'
+        oid: cpuUsage
+        precision: 1
+        type: saf-cfml4
+  sensors: 
+    state: 
+      data: 
+        - 
+          descr: 'Active link'
+          index: 'activeLink.{{ $index }}'
+          num_oid: '.1.3.6.1.4.1.7571.100.1.1.2.22.5.1.{{ $index }}'
+          oid: activeLink
+          state_name: activeLink
+          states: 
```

Comment:
Would you mind formatting these states as per other examples:

```
                    states:
                        - { value: 1, descr: other, graph: 1, generic: 3 }
                        - { value: 2, descr: unknown, graph: 1, generic: 3 }
                        - { value: 3, descr: ok, graph: 1, generic: 0 }
                        - { value: 4, descr: non-critical, graph: 1, generic: 1 }
                        - { value: 5, descr: critical, graph: 1, generic: 2 }
                        - { value: 6, descr: non-recoverable, graph: 1, generic: 2 }
```

Reply:
I changed the format of this yaml files as requested. It indeed looks better this way.